core/migrations/0077_unique_allocationsource_names_part2.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-04-04 03:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('core', '0076_unique_allocationsource_names_part1'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='allocationsource',
+            name='name',
+            field=models.CharField(editable=False, max_length=255, unique=True),
+        ),
+    ]
```

Comment:
Now that all those potential duplicates are gone (see migration 76), we can add the constraint.